StepLang.Tooling.Formatting.Tests/StepLang.Tooling.Formatting.Tests.csproj

```diff
@@ -21,10 +20,17 @@
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
             <PrivateAssets>all</PrivateAssets>
         </PackageReference>
+        <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
```

Comment:
technically, these changes don't fit the commit message. were they intended?

Reply:
Uhhh, right, the message does not fit. The change is intentional but should have gotten its own commit.